src/components/page/body.vue

```diff
@@ -22,11 +20,26 @@ export default {
   name: 'PageBody'
 };
 </script>
+<script setup>
+import { computed, inject } from 'vue';
+
+const { router } = inject('container');
+const htmlClass = computed(() => ({
+  // `router` may be `null` in testing.
```

Comment:
It's rare for a test to render a `PageBody` component but not inject a router. However, that happens in the tests of `AsyncRoute`.